### ISSUE TITLE
IsThere.sync in README.md

### ISSUE THREAD

**Issue by keharriso** [IonicaBizau/node-is-there#1]
In the example provided in the README file, you have:

``` javascript
// Sync call
var exists = IsThere("path/to/the/file/or/directory")
```

Unfortunately, this code raises an exception. Instead of IsThere, you should use IsThere.sync, making it clear that they are two separate functions.

Alternatively, it'd be neat if the code in the README really did work. You could check if the given callback is undefined, and if it is, call the sync version. This is obviously more a matter of taste.

**Comment by IonicaBizau** [comment]
Whoups, that's true! 

Also, the idea of making the callback optional is neat! :sparkles: Could you please fix this?

**Comment by IonicaBizau** [comment]
Fixed via #5 and #2. :fireworks:
